.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,15 @@
+name: "Validate Gradle Wrapper"
```

Comment:
What’s your argument to have it in it’s own workflow instead of adding it as a step to our existing workflow?

Reply:
separate badge